src/atomate2/common/jobs/gruneisen.py

```diff
@@ -0,0 +1,250 @@
+"""Jobs for Grueneisen-Parameter computations."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import phonopy
+from jobflow import Flow, Response, job
+from phonopy.api_gruneisen import PhonopyGruneisen
+from phonopy.phonon.band_structure import get_band_qpoints_and_path_connections
+from pymatgen.io.phonopy import (
+    get_gruneisen_ph_bs_symm_line,
+    get_gruneisenparameter,
+    get_pmg_structure,
+)
+from pymatgen.io.vasp import Kpoints
+from pymatgen.phonon.gruneisen import (
+    GruneisenParameter,
+    GruneisenPhononBandStructureSymmLine,
+)
+from pymatgen.phonon.plotter import GruneisenPhononBSPlotter, GruneisenPlotter
+from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+
+from atomate2 import SETTINGS
+from atomate2.common.schemas.gruneisen import GruneisenParameterDocument
+from atomate2.common.schemas.phonons import PhononBSDOSDoc
+
+if TYPE_CHECKING:
+    from pymatgen.core.structure import Structure
+
+    from atomate2.forcefields.jobs import ForceFieldStaticMaker
+    from atomate2.vasp.jobs.base import BaseVaspMaker
+
+
+@job
+def shrink_expand_structure(structure: Structure, perc_vol: float) -> Response:
+    """
+    Create structures with expanded and reduced volumes.
+
```

Comment:
Docstrings not formatted according to numpy style, also missing `Parameters` title. Noticed this is the case for many functions in this file so won't mention it again.

Reply:
Thanks, will fix. Overlooked those when I started from the pull request ;).

Reply:
Done